region/lowernorfair/east.json

```diff
@@ -1192,7 +1191,41 @@
           "to": [
             {
               "id": 4,
-              "requires": [ "canUsePowerBombs" ],
+              "requires": [ 
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},                
+                "canUsePowerBombs",
+                {"or": [
+                  "canWalljump",
+                  {"cost": [
+                    {"heatFrames": 200}
+                  ]}
+                ]},
+                {"and" :[
+                  {"cost": [
+                    {"heatFrames": 800}
+                  ]},
+                  {"or": [
+                    {"cost": [
+                      {"enemyDamage": [{
+                        "enemy": 79,
+                        "hits": 1
+                      }]}
+                    ]},
+                    "ScrewAttack",
+                    "Super",
+                    {"and": [
+                      "Plasma",
+                      {"or": [
+                        "Ice",
```

Comment:
I see Plasma was rejected as an option here. I'm guessing that's on the basis that it takes 6 hits instead of 4 and that's the line where avoiding hits becomes unlikely?

Reply:
Yes at some point many of these rooms just become kind of a drag if you need to stay around and avoid everything.

Reply:
On the one hand I think killing the one Dessgeega at 4 with just Plasma and getting to PBs of shame is reasonable to have in logic, but I can see the argument for just having one hard req for killing Dessgeegas while heatrunning. So this is fine IMO.